### ISSUE TITLE
@decompile/examine with large attribute contents

### ISSUE THREAD

**Issue by brazilofmux** [brazilofmux/tinymux#217]
Original [issue 214](https://code.google.com/p/tinymux/issues/detail?id=214) created by brazilofmux on 2006-11-15T16:04:09.000Z:

<b>What steps will reproduce the problem?</b>
1. Fill an attribute's contents, end it with a unique character
2. ex #obj/attribute or @decompiel #obj/attribute.  It'll show cutoff.

<b>What is the expected output? What do you see instead?</b>
Should see the entire contents, but the contents will be cut off.

<b>Please use labels and text to provide additional information.</b>

This actually is not a bug.  It's a feature of the codebase.  However, this
may not be expected behavior when you're @decompiling objects to port to
other places.  The issue is that the attribute name, the attribute flags,
the attribute highlighting and the attribute contents are all contained in
a single LBUF.  The larger the attribute name, and the more flags on the
attribute displayed, the less of the attribute contents will be displayed.
 This, for obvious reasons, may not be the preferred output.

My suggestion is to have seperate switches to @decompile and examine that
basically double-buffers the output.  one for the attribute name, flags,
and highlight, one for the attribute contents.  That way, when using the
switch, the entirity of the decompile or examine can be seen.

Why not make it the default?  It may break backward compatability, and it's
not really a bug.
